app.py

```diff
@@ -4,33 +4,22 @@
 from flask.views import MethodView
 from werkzeug.exceptions import HTTPException
 from flasgger import Swagger, swag_from
-from error_handling import CustomError, ValidationError, WrongIdError
+from error_handling import CustomError, WrongIdError
+from marshmallow import Schema, fields, ValidationError
 
 
-class Validation:
-    pattern = {'name': str, 'address': str, "work_time": str, "phone_number": str}
+class RestaurantSchemaPost(Schema):
```

Comment:
there can be one schema: `RestaurantCreateOrUpdateSchema`, but in `PUT` partial loading can be used:
https://marshmallow.readthedocs.io/en/stable/quickstart.html#partial-loading

Reply:
fixed